src/main/scala/com/microsoft/partnercatalyst/fortis/spark/StreamProviderFactory.scala

```diff
@@ -0,0 +1,43 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+import java.io.File
+
+import com.microsoft.partnercatalyst.fortis.spark.streamfactories.adapters.TadawebAdapter
+import com.microsoft.partnercatalyst.fortis.spark.streamfactories._
```

Comment:
Not a huge fan of wildcard imports since they have [a few negatives](http://seanmonstar.com/post/708954358/import-star-considered-harmful) and don't add much value in the times of IDE-driven auto-imports.